mmv1/products/cloudrun/api.yaml

```diff
@@ -616,6 +616,68 @@ objects:
+            - !ruby/object:Api::Type::NestedObject
+              name: startupProbe
+              min_version: beta
+              description: |-
+                Startup probe of application within the container.
+                All other probes are disabled if a startup probe is provided, until it
+                succeeds. Container will not be added to service endpoints if the probe fails.
+              properties:
+                - !ruby/object:Api::Type::Integer
+                  name: initialDelaySeconds 
+                  description: |-
+                    Number of seconds after the container has started before the probe is
+                    initiated.
+                    Defaults to 0 seconds. Minimum value is 0. Maximum value is 240.
+                - !ruby/object:Api::Type::Integer
+                  name: timeoutSeconds 
+                  description: |-
+                    Number of seconds after which the probe times out.
+                    Defaults to 1 second. Minimum value is 1. Maximum value is 3600.
+                    Must be smaller than periodSeconds.
+                - !ruby/object:Api::Type::Integer
+                  name: periodSeconds 
+                  description: |-
+                    How often (in seconds) to perform the probe.
+                    Default to 10 seconds. Minimum value is 1. Maximum value is 240.
+                - !ruby/object:Api::Type::Integer
+                  name: failureThreshold  
+                  description: |-
+                    Minimum consecutive failures for the probe to be considered failed after
+                    having succeeded. Defaults to 3. Minimum value is 1.
+                - !ruby/object:Api::Type::NestedObject
+                  name: tcpSocket 
+                  description: |-
+                    TcpSocket speficies an action involving a TCP port.
+                  properties:
+                    - !ruby/object:Api::Type::Integer
+                      name: port
+                      description: |-
+                        Port number to access on the container. Number must be in the range 1 to 65535.
+                - !ruby/object:Api::Type::NestedObject
+                  name: httpGet 
+                  description: |-
+                    HttpGet specifies the http request to perform..
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: path
+                      description: |-
+                        Path to access on the HTTP server.
+                    - !ruby/object:Api::Type::Array
+                      name: httpHeaders
+                      description: |-
+                        Custom headers to set in the request. HTTP allows repeated headers.
+                      item_type: !ruby/object:Api::Type::NestedObject
+                        properties:
+                          - !ruby/object:Api::Type::String
+                            name: name
```

Comment:
Same as above with both these subfields

Reply:
I'm not sure why `http_headers {}` causes test failing but not `http_headers {name = "somename"}` or `http_headers {value = "somevalue"}

Reply:
If that causes a test failure we should cover it with a test & work out how to get it to pass.

Is this a valid specification?

```
http_headers {
  name = ""
}
```

Reply:
Yes it's a valid specification. I tried a test for it but it failed due to

```
           ~ http_get {
                  # (1 unchanged attribute hidden)

                + http_headers {}

                  # (1 unchanged block hidden)
              }
```

Do you have any suggestions to fix it?

Reply:
This may need `send_empty_value: true` as well, per above!